api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -2429,6 +2429,12 @@
     <changeSet author="ben (generated)" id="1227303685425-75">
         <addPrimaryKey columnNames="concept_id, concept_set" tableName="concept_set_derived"/>
     </changeSet>
+    <changeSet author="vshankar" id="201609171146-1">
```

Comment:
I did try moving it to update-to-latest at the top before all the other changesets. I got this error message:

`Error executing SQL ALTER TABLE `cohort_member` DROP PRIMARY KEY: Error on rename of './openmrs/#sql-175b4_a' to './openmrs/cohort_member' (errno: 150 - Foreign key constraint is incorrectly formed): Caused By: Error executing SQL ALTER TABLE `cohort_member` DROP PRIMARY KEY: Error on rename of './openmrs/#sql-175b4_a' to './openmrs/cohort_member'`